src/setup_payload/SetupPayload.h

```diff
@@ -89,10 +89,10 @@ const char * const kQRCodePrefix = "MT:";
 /// The rendezvous type this device supports.
 enum class RendezvousInformationFlag : uint8_t
 {
-    kNone      = 0,      ///< Device does not support any method for rendezvous
-    kSoftAP    = 1 << 0, ///< Device supports Wi-Fi softAP
-    kBLE       = 1 << 1, ///< Device supports BLE
-    kOnNetwork = 1 << 2, ///< Device supports Setup on network
+    kSoftAP    = 0, ///< Device supports Wi-Fi softAP
```

Comment:
Are you sure this is correct? I looked at `Discovery Capabilities Bitmask` in the spec and the previous version was correct.

Reply:
Yes, sorry for the confusion the previous version was correct